### DISCUSSION TITLE
/DBC-Wiki/en/staking-model/generate-new-account.html

### DISCUSSION THREAD

**Discussion by giscus** [DeepBrainChain/DBC-Wiki#38] [Announcements]
# /DBC-Wiki/en/staking-model/generate-new-account.html

Welcome to DBC-Wiki

https://deepbrainchain.github.io/DBC-Wiki/en/staking-model/generate-new-account.html

**Comment by abydongroup** [comment]
guys, we need a Ledger based wallet
